scopes/react/react/webpack/webpack.config.base.ts

```diff
@@ -43,12 +33,10 @@ export default function (isEnvProduction = false): Configuration {
   // passed into alias object. Uses a flag if passed into the build command
   const isEnvProductionProfile = process.argv.includes('--profile');
 
-  const baseStyleLoadersOptions = {
-    injectingLoader: isEnvProduction ? MiniCssExtractPlugin.loader : styleLoaderPath,
-    cssLoaderPath: require.resolve('css-loader'),
-    postCssLoaderPath: require.resolve('postcss-loader'),
-    postCssConfig,
-  };
+  const { styleLoaders, stylePlugins } = makeStyleLoaders({
+    postcssOptions: postCssConfig,
+    styleInjector: isEnvProduction ? undefined : 'style-loader',
```

Comment:
what about the miniCssOptions for the plugin? you changed the behaviour of the plugin now

Reply:
pass as injector options